.vale.ini

```diff
@@ -8,7 +8,7 @@ StylesPath = .vale/styles
 
 # Vocab define the exceptions to use in *all* `BasedOnStyles`.
 # See: https://docs.errata.ai/vale/vocab
-; Vocab = RedHat
+Vocab = Base
```

Comment:
The [Installing Vale with the RedHat package](https://redhat-documentation.github.io/vale-at-red-hat/docs/user-guide/installing-vale-cli/) doc recommends this configuration, but the actual file sample has a Vocab disabled. Which is the correct configuration for enabling vale-at-red-hat?

Reply:
Let's standardize on `Vocab = Base`.

The Vocabulary is a very powerful configuration that overrides the style rules.

In this repository, as we deliver a style, we must keep it empty. 

In a project, you may need to define a Vocabulary to override rules that are in your styles, without editing the styles themselves. For instance, you can continue using the Red Hat package, but modify its entire behavior. It is good for quick results, but most of the time it's better to enhance the style.

https://vale.sh/docs/topics/vocab/

Reply:
```
E100 [vocab] Runtime error

'Base' does not exist
```

As we are here distributing a style, we don't have a Vocabulary in the repository.

To enable the Vocabulary, the required files need to be present. You need to create a `.vale/Vocab/Base` directory. To commit something in git, add empty `accept.txt` and `reject.txt` files.

I don't see this change as particularly relevant in this repository.

Reply:
Packages in https://vale.sh/hub/ are defined without Vocab.
The generator at https://vale.sh/generator/ is adding a Vocab line.

The Vocab being optional, I believe it's better to omit the Vocab line in our docs. Therefore remove it from https://github.com/redhat-documentation/vale-at-red-hat/blob/main/modules/user-guide/pages/installing-vale-cli.adoc.

Adding a Vocabulary is a topic for advanced users, and I believe in that case they should look at the upstream doc: https://vale.sh/docs/topics/vocab/.

Reply:
@themr0c - Fixed. Vocab line is now removed from the doc sample .vale.ini configuration in this PR.